docs/checkouts/features.mdx

```diff
@@ -79,15 +74,16 @@ _note: Disable the BNPL option by passing the correspondent prop to the web comp
 ```
 
 - The application fee applies based on the selected payment method during checkout.
-_note: available for Card, ACH and Terminal (Card Present) only!_
+  It overwrites the `application_fee_rate` set for the platform and/or sub account.
```

Comment:
I think the application_fees params only overwrite the sub account application_fee_rate, not the platform one. If my understanding is correct the platform application_fee_rate is contractually set and should not be available to be changed/overwritten via API.